packages/client/src/runtime/core/engines/client/ClientEngine.ts

```diff
@@ -42,25 +37,18 @@ export class ClientEngine implements Engine<undefined> {
   QueryCompilerConstructor?: QueryCompilerConstructor
   queryCompilerLoader: QueryCompilerLoader
 
-  executingQueryPromise?: Promise<any>
```

Comment:
ℹ️ In the `LibraryEngine` this was used to kinda await the last running query when stopping the engine. But as multiple queries can be in progress at a time I think this was never really accurate. As we no big stopping logic anymore it's imo not necessary to track our in progress queries like this.

If full query termination is really important to use we imo should rather look into using `AbortController` or so. 🤔